podman/domain/volumes.py

```diff
@@ -173,3 +174,49 @@ def remove(self, name: Union[Volume, str], force: Optional[bool] = None) -> None
             name = name.name
         response = self.client.delete(f"/volumes/{name}", params={"force": force})
         response.raise_for_status()
+
+    def export_archive(self, name: Union[Volume, str]) -> bytes:
+        """Export a podman volume, returns the exported archive as bytes.
+
+        Args:
+            name: Identifier for Volume to be exported.
+
+        Raises:
+            APIError: when service reports an error
+        """
+        if isinstance(name, Volume):
+            name = name.name
+        response = self.client.get(f"/volumes/{name}/export")
+        response.raise_for_status()
+        return response._content
+
+    def import_archive(
+        self, name: Union[Volume, str], data: Optional[bytes] = None, path: Optional[str] = None
```

Comment:
I was about to comment on this line about `|` before python 3.10, but you found it